src/background/auth/codeGrantFlow.ts

```
@@ -119,9 +120,12 @@ async function codeGrantFlow(
     redirect_uri,
     grant_type: "authorization_code",
     code,
-    client_id: params.client_id,
   };
 
+  if (client_id) {
+    tokenBody.client_id = client_id;
+  }
```

@johnnymetz @twschiller Isn't `client_id` technically required for the oauth2 auth server requests? Does it make sense from a practical standpoint that `client_id` would ever be missing, or should we throw an error here?